src/dune_engine/action_intf.ml

```diff
@@ -86,6 +86,7 @@ module Ext = struct
     { name : string
     ; version : int
     ; how_to_cache : Memoize_or_distribute.t
+    ; is_useful_to_sandbox : bool
```

Comment:
I guess `'a -> bool` and `'a -> Memoize_or_distribute.t` would be more consistent with the rest of the fields. (or if there's any reason for it being a constant, a comment could be useful)